grpcgcp/grpc_gcp/grpc_gcp.proto

```diff
@@ -17,21 +17,26 @@ syntax = "proto3";
 package grpc.gcp;
 
 message ApiConfig {
-  // The channel pool configurations.
+  // The channel/connection pool configurations.
   ChannelPoolConfig channel_pool = 2;
 
   // The method configurations.
   repeated MethodConfig method = 1001;
 }
 
 message ChannelPoolConfig {
```

Comment:
Could you add a general purpose comment? Something like,

> ChannelPoolConfig are options for configuring the channel pool.

Also, as part of this, it would be good to discuss the interaction between max_size and max_concurrent_streams_low_watermark. Maybe something like,

> RPCs will be scheduled onto existing channels until all channels have <max_concurrent_streams_low_watermark> number of streams. At this point a new channel is spun out and similarly filled with RPCs until <max_concurrent_streams_low_watermark>. Once <max_size> channels have been spun out, subsequent RPCs will hang until an in-flight RPC is finished, freeing up space.

(this is a very rough draft that I recommend you adjust as you see fit)

Reply:
Done.